File: existDB-frintend/src/app/view-manager/model/CreatedViewModel.ts
export class CreatedViewModel {
    user: string;
    date: string;
    configLocation: string;
    queryName: string;
    viewLocation: string;

    constructor(user: string, date: string, configLocation: string, queryName: string, viewLocation: string) {
        this.user = user;
        this.date = date;
        this.configLocation = configLocation;
        this.queryName = queryName;
        this.viewLocation = viewLocation;
    }
}

File: existDB-backend/src/main/java/hu/sule/administration/model/CreatedViewModel.java
package hu.sule.administration.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class CreatedViewModel {
    private String user;
    private String date;
    private String configLocation;
    private String queryName;
    private String viewLocation;
}
